inference-engine/tools/compile_tool/README.md

```diff
@@ -74,6 +79,20 @@ To dump a blob using a trained network, use the command below:
 You can compile executable network without a connected FPGA device with a loaded DLA bitstream.
 To do that, specify the architecture name of the DLA bitstream using the parameter `-DLA_ARCH_NAME`.
 
+## VPUX Plugin
+
+To compile a network using VPUX plugin, you need to pass `-d VPUX` parameter to `compile_tool`.
+If you have a VPU device connected, `compile_tool` will compile a network for it.
+
+If several VPU devices connected, the tool will choose the first enumerated device in a system.
+For example, if there are `VPUX.0-2490-A0` and `VPUX.1-2490-B0`, `compile_tool` will compile a network for `VPUX.0-2490-A0`.
+If you need to compile a network for `VPUX.1-2490-B0`, you can to do it explicitly by passing `-d VPUX.1-2490-B0`
+A list of VPUX devices available can be obtained by `hello_query_sample`. [TBD Link]
+
+You can compile network without a connected VPU device. To do that, specify the platform name of the device
```

Comment:
Is there any way to use '-d' option for the sake of compilation without connected device?
That will require less effort from the user to use single cmdline option.

Reply:
Good question :) 
It is a possible option. We are going to cover it on the meeting related to device naming.

Reply:
Should it be finalized and merged?